tooling/minimize/src/rvalue.rs

```diff
@@ -97,6 +97,21 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
 
                 ValueExpr::AddrOf { target, ptr_ty }
             }
+            smir::Rvalue::NullaryOp(null_op, ty) => {
+                let ty = smir::internal(self.tcx, ty);
+                let layout = self.rs_layout_of(ty);
+                match null_op {
+                    smir::NullOp::SizeOf =>
+                        build::const_int_typed::<usize>(translate_size(layout.size()).bytes()),
+                    smir::NullOp::OffsetOf(fields) =>
+                        build::const_int_typed::<usize>(self.translate_fields(fields, ty).bytes()),
```

Comment:
Why are you translating anything here? You can just call `offset_of_subfield` and then pass the result through `const_int_typed`